pkg/inputs/helper/binlog_checker/checker.go

```diff
@@ -162,13 +158,7 @@ func (checker *binlogChecker) IsEventBelongsToMySelf(row Row) bool {
 // update_time_at_source DATETIME(6) NOT NULL ON UPDATE CURRENT_TIMESTAMP
 // )
 func (checker *binlogChecker) MarkActive(row Row) {
```

Comment:
Let's add binlog checker back. We can rename the metrics and struct with a more general name like `SourceProbe...`

Reply:
added ProbeHistogram